hadoop-plugin/src/test/groovy/com/linkedin/gradle/azkaban/AzkabanDslYamlCompilerTest.groovy

```diff
@@ -79,6 +79,9 @@ class AzkabanDslYamlCompilerTest {
 
     Project mockProject = mock(Project.class);
     when(mockProject.name).thenReturn("test");
+    when(mockProject.getRootDir()).thenReturn(new File("/home/root_directory"))
```

Comment:
This path works on Linux but not on Mac.
/var/tmp/root_directory may work on both

Reply:
It's not meant to be a real path. won't be resolved